### DISCUSSION TITLE
Please make timeout infinity if debugging test

### DISCUSSION THREAD

**Discussion by hackwaly** [vitest-dev/vitest#6125] [Q&A] [answered]
Eg. In a debug session launched through codelens debug contextmenu item.

I think to specify testTimeout to workaround this issue is not a good idea.

**Comment by sheremet-va** [comment]
Do you mean when you run it in vscode debugger? Do you use the official extension?

**Reply by hackwaly** [reply]
Yes, I'm sure I'm using official extension.

**Reply by sheremet-va** [reply, marked as answer]
Should be fixed in https://github.com/vitest-dev/vscode/commit/0483a99cef39acd4c4c37744797e1df88a4eae40
